deploy/ansible/worker/README.md

```diff
@@ -3,24 +3,24 @@
 1. provision ubuntu hosts accessible from the internet and to which you can ssh into.
     * if you need to use a .pem file, use the "amazon" example (in /nucypher/deploy/ansible/worker/inventory.yml)
 
-2. use a locally running geth instance to create accounts for the workers you'll be deploying (you will need the keystores)
+2. if you would like to pre-create your worker accounts, use a locally running geth instance to create accounts for the workers you'll be deploying (you will need the keystores), otherwise worker accounts you can have workers created automatically as the nodes are deployed
```

Comment:
I don't understand the last part of the sentence here.

Reply:
Yeah it does sound a little awkward

Reply:
Will fix in the next PR!